lib/ReactViews/Map/Panels/SharePanel/SharePanel.jsx

```diff
@@ -397,8 +398,37 @@ const SharePanel = observer(
                 )
               }
             />
+            {/* Following code block dependent on existence of "bookmarkHelp" Help Menu Item */}
+            <If
+              condition={this.props.terria.configParameters.helpContent.some(
+                e => e.itemName === "bookmarkHelp"
+              )}
+            >
```

Comment:
We use less of `If` nowadays because it is not available for `tsx`. The standard react way of doing this is `{condition && <Text>...</Text>}`

Reply:
Resolved